app/templates/merchant/email/confirm_order.txt

```diff
@@ -0,0 +1,15 @@
+Hello!
```

Comment:
.TXT versions of emails are actually used more often than i thought :+1: especially on slow networks, which RTM may be (its best to be safe). plus this allows us to be more fancy with our regular HTML emails (we have the .txt fallback). If you want to test this in the future, i know at least the seas webmail has an option to view .txt versions of each email 

[See this picture for reference](http://puu.sh/n9GMs/9abe5c5a61.png)

Fixed for the other emails as well